src/main/scala/dpla/ingestion3/mappers/providers/Extractor.scala

```diff
@@ -9,4 +10,14 @@ import dpla.ingestion3.model.{DplaMapData, EdmAgent}
 trait Extractor {
   def build(rawData: String): DplaMapData
   def agent: EdmAgent
-}
+  /**
+    * Generates an MD5 checksum from the given providerId
+    * Implemented per provider
+    *
+    * DigestUtils.md5Hex(id)
+    *
+    * @return MD5 hash value
+    * @throws NotImplemented
+    */
+  def mintItemId(rawData: Any): String
```

Comment:
To clarify, when I say String, I mean literally the thing to hash, not the whole record.